package.json

```diff
@@ -22,39 +19,62 @@
   ],
   "scripts": {
     "clean": "rimraf ./dist",
-    "build": "babel ./src --out-dir ./dist",
+    "build": "babel ./src --out-dir ./dist --extensions .ts --extensions .js --copy-files",
     "prepublish": "npm run clean && npm run build",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
+    "lint": "eslint --cache --cache-location=.cache/eslint --ext .js,.jsx,.json,.html,.ts,.tsx,.mjs --report-unused-disable-directives",
     "storybook": "start-storybook -p 6006",
     "start": "concurrently \"npm run storybook -- --no-manager-cache --quiet\" \"npm run build -- --watch\"",
     "build-storybook": "build-storybook",
     "release": "npm run build && auto shipit --base-branch=main"
   },
-  "dependencies": {},
   "devDependencies": {
     "@babel/cli": "^7.12.1",
     "@babel/core": "^7.12.3",
     "@babel/preset-env": "^7.12.1",
     "@babel/preset-react": "^7.12.5",
-    "@storybook/react": "^6.1.14",
+    "@babel/preset-typescript": "^7.12.5",
+    "@storybook/addon-essentials": "^6.1.20",
+    "@storybook/addon-knobs": "^6.1.20",
+    "@storybook/addon-storyshots": "^6.1.20",
+    "@storybook/eslint-config-storybook": "^2.4.0",
+    "@storybook/svelte": "^6.1.20",
     "auto": "^10.3.0",
+    "babel-jest": "^26.6.3",
     "babel-loader": "^8.1.0",
     "chromatic": "^5.6.1",
     "concurrently": "^5.3.0",
+    "eslint": "^7.17.0",
+    "jest": "^26.6.3",
+    "jest-environment-jsdom-thirteen": "^1.0.1",
+    "prettier": "^2.2.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "rimraf": "^3.0.2"
+    "rimraf": "^3.0.2",
+    "svelte": "^3.32.3",
+    "svelte-jester": "^1.3.0",
+    "svelte-loader": "^2.9.1",
+    "typescript": "^3.3.3333"
   },
   "peerDependencies": {
-    "@storybook/addons": "^6.1.14",
-    "@storybook/api": "^6.1.14",
-    "@storybook/components": "^6.1.14",
-    "@storybook/core-events": "^6.1.14",
-    "@storybook/theming": "^6.1.14",
+    "@storybook/addons": "^6.1.20",
+    "@storybook/api": "^6.1.20",
+    "@storybook/client-api": "^6.1.20",
+    "@storybook/client-logger": "^6.1.20",
+    "@storybook/components": "^6.1.20",
+    "@storybook/core-events": "^6.1.20",
+    "@storybook/svelte": "^6.1.20",
+    "@storybook/theming": "^6.1.20",
     "react": "^16.8.0 || ^17.0.0",
-    "react-dom": "^16.8.0 || ^17.0.0"
+    "react-dom": "^16.8.0 || ^17.0.0",
+    "svelte": "^3.32.3",
+    "svelte-loader": "^2.9.1",
+    "ts-dedent": "^2.0.0"
```

Comment:
`ts-dedent` -> regular dep instead of peer?

Reply:
I'll fix that